tests_tf/test_mnist_tutorial_tf.py

```diff
@@ -0,0 +1,21 @@
+from inspect import getsourcefile
+import os
+import sys
+
+# Import the tutorial after evaluating its relative path
```

Comment:
Is the issue here that the tutorials aren't a submodule of `cleverhans` and there is probably something else called `tutorials` on the `PYTHONPATH`? Would it be easier to just rename them `cleverhans_tutorials` so they can be imported without changing the path?